be/src/exec/schema_scanner/schema_task_runs_scanner.cpp

```diff
@@ -135,29 +136,42 @@ Status SchemaTaskRunsScanner::fill_chunk(ChunkPtr* chunk) {
             break;
         }
         case 6: {
-            // DATABASE
+            // CATALOG
```

Comment:
is there any better way? I'm afraid it breaks compatibility for some tools.

Reply:
I don't think it will affect any tool, unless that tool only uses `select *`, and then processes it in column order, not by column name.